### PR TITLE
feat(index.html): improvement of UI

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Brush up the UI added in v3.0 and modernize the title bar.

## How Has This Been Tested?

- Check the appearance in light and dark modes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -360,6 +361,11 @@ function launch() {
     minHeight: 240,
     show: false,
     icon: icon,
+    titleBarStyle: 'hidden',
+    titleBarOverlay: {
+      color: nativeTheme.shouldUseDarkColors ? '#2e2e2e' : '#f0f2f4',
+      symbolColor: nativeTheme.shouldUseDarkColors ? '#FFF' : '#000',
+    },
```
Comment (hal-shu-sato) [inline src/main.ts:368]: I'd like to suggest changing `titleBarOverlay` depending on the OS setting.
I suggest adding:

- `update` event emit
  - https://www.electronjs.org/docs/latest/api/native-theme#event-updated
- `setTitleBarOverlay` according to `shouldUseDarkColors`
  - https://www.electronjs.org/docs/latest/api/browser-window#winsettitlebaroverlayoptions-windows
  - https://www.electronjs.org/docs/latest/api/native-theme#nativethemeshouldusedarkcolors-readonly

**Review by hal-shu-sato — COMMENTED** [review]
I have one suggestion of adding.

File: src/renderer/main/index.html [hunk 2]
```diff
@@ -12,11 +12,12 @@
 		<meta name="color-scheme" content="light dark" />
 	</head>
 
-	<body class="pt-1">
-		<nav>
+	<body>
+		<nav class="bg-light draggable">
+			<div class="titleBar app-name text-center text-muted"></div>
```
Comment (hal-shu-sato) [inline src/renderer/main/index.html:17]: Align center.
```suggestion
			<div class="title-bar app-name d-flex justify-content-center align-items-center text-muted"></div>
```

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
I have one more suggestion.
And, please check the naming convention of HTML and CSS, kebab-case.

**Comment by mitosagi** [comment]
I have fixed it.

File: src/renderer/main/index.css [hunk 3]
```diff
@@ -10,9 +13,21 @@ section#nicommons .container-lg > .card > .card-body {
...
+  height: calc(
+    100% - 2.625rem -
+      env(titlebar-area-height, var(--fallback-title-bar-height))
+  );
   overflow-y: auto;
 }
+.draggable {
+  -webkit-app-region: drag;
+}
+.nonDraggable {
```
Comment (hal-shu-sato) [inline src/renderer/main/index.css:25]:
```suggestion
.non-draggable {
```

**Review by hal-shu-sato — CHANGES_REQUESTED** [review]
Could you change here like this?

**Review by hal-shu-sato — APPROVED** [review]
It's a great design!
apm become cooler!!
Thank you for changing it and many fixing!!

**Pull request merged**